src/components/language/language.service.ts

```diff
@@ -499,13 +639,62 @@ export class LanguageService {
     _languageId: string,
     _locationId: string,
     _session: ISession
-  ): Promise<void> {}
+  ): Promise<void> {
+    const locationLabel = await this.getLocationLabelById(_locationId);
+
+    if (!locationLabel) {
+      throw new BadRequestException('Cannot find location');
+    }
+
+    await this.removeLocation(_languageId, _locationId, _session);
+    await this.db
+      .query()
+      .matchNode('language', 'Language', { id: _languageId, active: true })
+      .matchNode('location', locationLabel, {
+        id: _locationId,
+        active: true,
+      })
+      .create([
+        node('language'),
+        relation('out', '', 'location', {
+          active: true,
+          createdAt: DateTime.local(),
+        }),
+        node('location'),
+      ])
+      .run();
+  }
 
   async removeLocation(
     _languageId: string,
     _locationId: string,
     _session: ISession
```

Comment:
These were just prefixed with _ to make the linter happy. They can be renamed as soon as they are used.